### ISSUE TITLE
how to run my own dataset

### ISSUE THREAD

**Issue by zhangbeile** [WangYueFt/dcp#14]

**Comment by WangYueFt** [comment]
You probably need to rewrite or modify the dataloader, then you're ready to train on your own data.

**Comment by zhangbeile** [comment]
Hi Wang
&nbsp; I have a question how to test my own datasets like I have two ply I want to register one ply&nbsp;
Thanks&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by WangYueFt** [comment]
You could load the two ply and convert them to pytorch tensor. But I think there is work to do.

**Comment by zhangbeile** [comment]
thanks

**Comment by zhangbeile** [comment]
so I can use your code to register two ply file to one ply file right?I just change the two ply file convert them to pytorch tensor and register to one file why do i need label?if I need use v2 to test

**Comment by WangYueFt** [comment]
You could try but I am afraid you still have to some coding working.

Best,
Yue

**Comment by fellenB** [comment]
Hello WangYue，I am an engineer form China. I am very excited about your work and i want to use it in my robotic grasp project. Do you have demo code to test two ply data ? or can you give me some detail advice about how to modify your code to test two ply data?
thank you very much!
